File: 프로그래머스/Lv.1/12932. 자연수 뒤집어 배열로 만들기/자연수 뒤집어 배열로 만들기.cs
using System.Collections.Generic;
public class Solution {
    public int[] solution(long n) {
        int[] answer = new int[] {};
        List<int> arr = new List<int>();
        long x =1;
        long temp;
        while(n/x != 0)
        {
            temp = (n/x);
            arr.Add((int)(temp%10));
            x = x* 10;
        }
        answer = arr.ToArray();
        return answer;
    }
}

File: 프로그래머스/1/12932. 자연수 뒤집어 배열로 만들기/자연수 뒤집어 배열로 만들기.java
import java.util.List;
import java.util.ArrayList;

class Solution {
    public int[] solution(long n) {
        List<Integer> arr = new ArrayList<>();
        long i = 1;
        while(n / i > 0)
        {
            arr.add((int)((n / i) % 10));
            i *= 10;
        }
        
        int[] answer = arr.stream()
            .mapToInt(Integer::intValue)
            .toArray();
        return answer;
    }
}